resolwe_bio/processes/clustering/pca.yml

```diff
@@ -45,18 +45,18 @@
       {% endfor %}
 
       {% for exp in exps %}
-        exp=$exp"{{ exp.exp.file }},"
-        obj_id=$obj_id"{{ exp|sample_id }},"
+        exp_files=$exp_files"{{ exp.exp.file }},"
+        sample_ids=$sample_ids"{{ exp|sample_id }},"
       {% endfor %}
 
       re-save proc.rc $?
       re-save proc.progress 0.1
 
-      obj_id=${obj_id%?}
-      exp=${exp%?}
+      sample_ids=${sample_ids%?}
+      exp_files=${exp_files%?}
 
       re-save proc.rc $?
       re-save proc.progress 0.2
 
```

Comment:
For clustering it we added checking if is the source of selected genes the same as the source of expression files, dose it make sense to add this here?